### PR TITLE
New package: Proton.ProtonMail version 0.9.3

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138622)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping Exorcism0666.
>[!warning]
>With the new version of Komac, numerous Pull Requests may contain bugs or unwanted elements. If such issues arise, please tag russellbanks so that these bugs can be addressed in future updates.

>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/p/Proton/ProtonMail/0.9.3/Proton.ProtonMail.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Proton.ProtonMail
```
Comment (Trenly) [inline manifests/p/Proton/ProtonMail/0.9.3/Proton.ProtonMail.installer.yaml:4]:
```suggestion
PackageIdentifier: Proton.ProtonMail.Beta
```

File: manifests/p/Proton/ProtonMail/0.9.3/Proton.ProtonMail.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Proton.ProtonMail
```
Comment (Trenly) [inline manifests/p/Proton/ProtonMail/0.9.3/Proton.ProtonMail.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Proton.ProtonMail.Beta
```

File: manifests/p/Proton/ProtonMail/0.9.3/Proton.ProtonMail.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Proton.ProtonMail
```
Comment (Trenly) [inline manifests/p/Proton/ProtonMail/0.9.3/Proton.ProtonMail.yaml:4]:
```suggestion
PackageIdentifier: Proton.ProtonMail.Beta
```

File: manifests/p/Proton/ProtonMail/0.9.3/Proton.ProtonMail.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Proton.ProtonMail
+PackageVersion: 0.9.3
+InstallerType: portable
```
Comment (Trenly) [inline manifests/p/Proton/ProtonMail/0.9.3/Proton.ProtonMail.installer.yaml:6]:
```suggestion
InstallerType: exe
```

File: manifests/p/Proton/ProtonMail/0.9.3/Proton.ProtonMail.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Proton.ProtonMail
+PackageVersion: 0.9.3
+PackageLocale: en-US
+Publisher: Proton AG
+PackageName: Proton official desktop application for Proton Mail and Proton Calendar
```
Comment (Trenly) [inline manifests/p/Proton/ProtonMail/0.9.3/Proton.ProtonMail.locale.en-US.yaml:8]:
```suggestion
PackageName: Proton Mail Beta
```
This should match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Pull request closed without merging**